com.github.weclaw1.ScoreTracker.yaml

```
@@ -0,0 +1,33 @@
+app-id: com.github.weclaw1.ScoreTracker
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: score-tracker
+finish-args:
+- --share=ipc
+- --socket=wayland
+- --socket=fallback-x11
+- --device=dri
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    CARGO_HOME: /run/build/score-tracker/cargo
+modules:
+- name: score-tracker
+  buildsystem: simple
+  build-commands:
+  - cargo --offline fetch --manifest-path Cargo.toml
+  - cargo --offline build --release
+  - install -Dm755 ./target/release/score-tracker -t /app/bin/
+  - install -Dm644 ./src/resources/com.github.weclaw1.ScoreTracker.svg -t /app/share/icons/hicolor/scalable/apps/
+  - install -Dm644 ./src/resources/com.github.weclaw1.ScoreTracker-symbolic.svg -t /app/share/icons/hicolor/symbolic/apps/
+  - install -Dm644 ./src/resources/com.github.weclaw1.ScoreTracker.desktop -t /app/share/applications/
+  - install -Dm644 ./src/resources/com.github.weclaw1.ScoreTracker.metainfo.xml -t /app/share/metainfo/
+  sources:
+  - cargo-sources.json
+  - type: git
+    url: https://github.com/weclaw1/score-tracker.git
+    tag: 1.0
```

```suggestion
    tag: '1.0'
```
Fixes a small warning

---

Fixed